test/unit/views/organisations/dataset-overview.test.js

```diff
@@ -0,0 +1,88 @@
+import { describe, it, expect } from 'vitest'
+import nunjucks from 'nunjucks'
```

Comment:
Better way of using nunjucks in our tests is like this: 

```javascript
import { setupNunjucks } from '../../src/serverSetup/nunjucks.js'
const nunjucks = setupNunjucks({ datasetNameMapping: new Map() })
```

This will make sure the data you're passing in is validated agains the schema.

See `check-answers.test.js`, for example.

Reply:
Thanks Done!